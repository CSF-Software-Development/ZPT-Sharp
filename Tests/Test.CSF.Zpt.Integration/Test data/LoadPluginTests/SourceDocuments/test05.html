<html>
<head>
<title>Load plugin test 5</title>
</head>
<body>
<div>
  <ul tal:define="myMacro here/Documents/test05/macros/test2">
    <li tal:repeat="item here/Items">
      <div tal:define="doc load:myMacro"
           tal:replace="structure doc">
        This content should be replaced by the rendered document
      </div>
    </li>
  </ul>
</div>
<div class="macros" tal:condition="nothing">
  <div metal:define-macro="test1">
    <p>Content from the macro, before the slot-filler</p>
    <p metal:define-slot="first">Default slot content here</p>
    <p>Content from the macro, after the slot-filler</p>
  </div>
  <div metal:extend-macro="test1"
       metal:define-macro="test2">
    <p>This is content from a different macro</p>
    <p metal:fill-slot="first">
      This is slot content from macro 'test2'.
      The item name is <strong tal:content="item/Name | default">Empty</strong>.
    </p>
    <p>This is content from a different macro</p>
  </div>
</div>
</body>
</html>