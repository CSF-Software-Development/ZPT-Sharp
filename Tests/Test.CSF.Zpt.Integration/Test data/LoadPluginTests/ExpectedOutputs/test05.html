<html>
<head>
<title>Load plugin test 5</title>
</head>
<body>
<div>
  <ul>
    <li>
      <div>
    <p>Content from the macro, before the slot-filler</p>
    <p>
      This is slot content from macro 'test2'.
      The item name is <strong>One</strong>.
    </p>
    <p>Content from the macro, after the slot-filler</p>
  </div>
    </li>
    <li>
      <div>
    <p>Content from the macro, before the slot-filler</p>
    <p>
      This is slot content from macro 'test2'.
      The item name is <strong>Two</strong>.
    </p>
    <p>Content from the macro, after the slot-filler</p>
  </div>
    </li>
    <li>
      <div>
    <p>Content from the macro, before the slot-filler</p>
    <p>
      This is slot content from macro 'test2'.
      The item name is <strong>Three</strong>.
    </p>
    <p>Content from the macro, after the slot-filler</p>
  </div>
    </li>
    
  </ul>
</div>

</body>
</html>